### PR TITLE
Improve log_url by requiring context data to be specified

### PR DESCRIPTION
**Pull request by alexjurkiewicz**

### CODE REVIEW

**Comment by alexjurkiewicz** [comment]
This bumps the major version, since a new required parameter is added.

It would be easy to make the parameter optional, and to fall back to the current log_url if unset. But I think it's simpler to add it as required.

What do you think?

**Comment by alexjurkiewicz** [comment]
PS, if you accept this as-is, I would also like to propose removing the default from `status`. I added this in #1 😊 but now I think it's a mistake -- there's no obvious default for this value so it looks confusing to people reading a workflow.

Since removing the default is a major update, it could be bundled into the 1.0.0 version bump.

File: entrypoint.sh [hunk 1]
```diff
@@ -9,8 +9,14 @@ get_from_event() {
   jq -r "$1" "${GITHUB_EVENT_PATH}"
 }
 
+# Test required inputs are set
+if [ -z "$INPUT_RUN_ID" ]; then
+  echo "Missing input run_id"
+fi
```
Comment (rsotnychenko) [inline entrypoint.sh]: @alexjurkiewicz shouldn't we fail the script here, since the run id variable is marked as required?

**Review by rsotnychenko — COMMENTED** [review]
Thanks for the contribution, @alexjurkiewicz!

My apologies for the delay. Everything looks great, apart from one minor comment.

I totally agree with removing the default value from `status`, but I'm not sure whether we should make it a 1.0 release until the underlying GitHub API is marked as stable.
Fortunately, SemVer spec allows us to do any breaking changes with a minor version number increment if the major version is equal to 0. So I'm thinking of releasing these changes as `0.2.0`. What do you think?

File: entrypoint.sh (hunk 1 above)
Comment (alexjurkiewicz) [reply entrypoint.sh]: You are right. I'll fix this.

**Comment by alexjurkiewicz** [comment]
Thanks for your feedback.

About semver, I don't have strong feelings. There is no mechanism to auto-update the versions of actions in a workflow, and there's not even mechanism to know the version in use is old. So I don't imagine many people upgrade action versions once a workflow is built.

I guess the question is: for anyone who _does_ decide to upgrade their version of this action, what version number would make their life easiest?

**Comment by alexjurkiewicz** [comment]
I've addressed your review feedback and changed the version to 0.2.0.

**Comment by alexjurkiewicz** [comment]
Ping <3

**Comment by alexjurkiewicz** [comment]
I've added two new commits:

* Make `status` input required again, as discussed.
* Add optional input `deployment_url`. This allows you to edit deployment status from a non-deployment workflow fun. For example, when `on: repository_dispatch`

**Comment by rsotnychenko** [comment]
@alexjurkiewicz many thanks! :)
I'll try to review and merge everything tomorrow

**Comment by rsotnychenko** [comment]
Thanks a lot! Everything is merged and released

**Pull request closed without merging**